01_01_Basic_Rules.txt

```diff
@@ -515,7 +515,7 @@ forces a recalculation on all affected values.
 +-------+----------+-------+------------+------------+---------------+
 | 1     | Static   |       |            |            |               |
 +-------+----------+-------+------------+------------+---------------+
-| 2     | 2*int    | 2     |            |            | 2             |
+| 2     | 2*int/cha| 2     |            |            | 2             |

```

Comment:
This is a needed update, but "int/cha" might be read as pick one...and then next time pick one (not necessarily the same). It's not likely to actually come up since it should be rare for a character to go from int > cha to cha > int, but could we try to fit "2*Skill Point Gain", naming the secondary stat explicitly?

Reply:
cool beans. Much better than the /

Reply:
done